Tella/Scenes/Settings/Views/Servers/ServersListView.swift

```diff
@@ -74,15 +75,21 @@ struct ServersListView: View {
         }
     }
     
-    private func handleActions(item: ListActionSheetItem) {
+    private func handleActions(item: ListActionSheetItem, server: Server) {
         guard let type = item.type as? ServerActionType else { return  }
         
         switch type {
         case .edit:
+            if (server.serverType == ServerConnectionType.uwazi.rawValue) {
```

Comment:
What do you think to change it to 

         private func handleActions(item: ListActionSheetItem, server: Server) {
        
        guard let type = item.type as? ServerActionType else { return  }
        switch type {
        case .edit:
            showEditServer()
        case .delete:
            showDeleteServerConfirmationView(server: server)
        }
        sheetManager.hide()
    }
    
    private func showEditServer() {
        guard let server = serversViewModel.currentServer else { return }
        
        switch server.serverType {
        case .tella:
            shouldShowEditServer = true
        case .uwazi:
            navigateTo(destination: UwaziAddServerURLView(appModel: mainAppModel, server: server)
                .environmentObject(serversViewModel))
        default:
            break
        }
        
    }

Reply:
Done